CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing
+
+This guide is meant to help you start contributing to the Symfony CLI by
+providing some key hints and explaining specifics related to this project.
+
+## Language choice
+
+First-time contributors could be surprised by the fact that this project is
+written in Go whereas it is highly related to the Symfony Framework which is
+written in PHP.
+
+Go has been picked because it is well suited for system development and has
+close-to-zero runtime dependencies which make releasing quite easy. This is
+ideal for a tool that is used on a wide range of platforms and potentially on
+systems where the requirements to run Symfony are not met. Go is also usually
+quite easy to apprehend for PHP developers having some similarities in their
+approach.
```

Comment:
Another reason is to make Symfony CLI independant of the PHP version (as one goal of the CLI is to make it possible to use many different versions of PHP).

Reply:
Forgot about this one 😄 Added